docs/en/solutions/apply_catboost_model.md

```diff
@@ -0,0 +1,204 @@
+# Applying a CatBoost model in ClickHouse {#applying-catboost-model-in-clickhouse}
+
+[CatBoost](https://catboost.ai) — is a free and open-source gradient boosting library for machine learning.
+
+To apply a CatBoost model in ClickHouse:
+
+1. [Create a table](#create-table).
+2. [Insert the data to the table](#insert-the-data-to-the-table).
+3. [Configure the model](#configure-the-model).
+4. [Run the model inference from SQL](#run-the-model-inference).
+
+For more information about training CatBoost models, see [Training and applying models](https://catboost.ai/docs/features/training.html#training).
+
+## Before you start {#before-you-start}
```

Comment:
as far as I remember we normally try to avoid "you", so it could be just prerequisites or something